test/functional/feature_asmap.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test asmap config argument for ASN-based IP bucketing.
+
+Verify node behaviour and debug log when launching dashd in these cases:
+
+1. `dashd` with no -asmap arg, using /16 prefix for IP bucketing
+
+2. `dashd -asmap=<absolute path>`, using the unit test skeleton asmap
+
+3. `dashd -asmap=<relative path>`, using the unit test skeleton asmap
+
+4. `dashd -asmap/-asmap=` with no file specified, using the default asmap
+
+5. `dashd -asmap` with no file specified and a missing default asmap file
+
+6. `bitcoind -asmap` with an empty (unparsable) default asmap file
```

Comment:
```suggestion
6. `dashd -asmap` with an empty (unparsable) default asmap file
```

Reply:
The pitfalls on not looking into _all commits_ that modify a file 😿. Resolved in 267426d180f9896f73a4a34c9365835549f11109